content/zh/blog/2019/app-identity-and-access-adapter/index.md

```diff
@@ -1,64 +1,61 @@
 ---
-title: App Identity and Access Adapter
-subtitle: Using Istio to secure multi-cloud Kubernetes applications with zero code changes
-description: Using Istio to secure multi-cloud Kubernetes applications with zero code changes.
+title: APP 身份和访问适配器
+subtitle: 使用 Istio 零代码实现多云 Kubernetes 应用安全
+description: 使用 Istio 零代码实现多云 Kubernetes 应用安全。
 publishdate: 2019-09-18
 attribution: Anton Aleksandrov (IBM)
 keywords: [security,oidc,jwt,policies]
 target_release: 1.3
 ---
 
-If you are running your containerized applications on Kubernetes, you can benefit from using the App Identity and Access Adapter for an abstracted level of security with zero code changes or redeploys.
+如果在 Kubernetes 以容器化的方式运行应用，就可以使用 App 身份和访问适配器获得抽象级别的安全性，而无需更改或重新部署代码。
 
-Whether your computing environment is based on a single cloud provider, a combination of multiple cloud providers, or following a hybrid cloud approach, having a centralized identity management can help you to preserve existing infrastructure and avoid vendor lock-in.
+无论您的运行环境是单云提供商，多个云提供商的组合，还是遵循混合云方法，集中式身份管理都可以帮助您维护现有基础设施并避免供被应商绑定。
 
-With the [App Identity and Access Adapter](https://github.com/ibm-cloud-security/app-identity-and-access-adapter), you can use any OAuth2/OIDC provider: IBM Cloud App ID, Auth0, Okta, Ping Identity, AWS Cognito, Azure AD B2C and more. Authentication and authorization policies can be applied in a streamlined way in all environments — including frontend and backend applications — all without code changes or redeploys.
+有了 [App 身份和访问适配器](https://github.com/ibm-cloud-security/app-identity-and-access-adapter)，就可以使用任何 OAuth2/OIDC 提供商：IBM Cloud App ID, Auth0, Okta, Ping Identity, AWS Cognito, Azure AD B2C 和其它更多。身份和授权策略可以以高效的方式应用在所有环境（包括前端和后端应用程序），而无需修改代码或重新部署。
 
-## Understanding Istio and the adapter
+## 了解 Istio 和其适配器 {#understanding-istio-and-the-adapter}
```

Comment:
#understanding-istio-and-the-adapter
->
#understanding-Istio-and-the-adapter

Reply:
accept